pyhap/accessory_driver.py

```diff
@@ -626,16 +626,30 @@ def get_characteristics(self, char_ids):
         :rtype: dict
         """
         chars = []
-        for id in char_ids:
-            aid, iid = (int(i) for i in id.split('.'))
-            rep = {HAP_REPR_AID: aid, HAP_REPR_IID: iid}
-            char = self.accessory.get_characteristic(aid, iid)
+        for aid_iid in char_ids:
+            aid, iid = (int(i) for i in aid_iid.split("."))
+            rep = {
+                HAP_REPR_AID: aid,
+                HAP_REPR_IID: iid,
+                HAP_REPR_STATUS: SERVICE_COMMUNICATION_FAILURE,
```

Comment:
I wonder if it'd make sense to use some other code to render the accessory completely unavailable from HomeKit's perspective (see [comment](https://github.com/homebridge/HAP-NodeJS/issues/499#issuecomment-335736097)).

Reply:
That is exactly what is going on here (completely unavailable to homekit)

<img width="292" alt="Screen Shot 2020-05-17 at 9 54 18 AM" src="https://user-images.githubusercontent.com/663432/82152047-6b9e4300-9824-11ea-95ff-b56c2441c175.png">

Reply:
Great, thanks for clarifying.